internal/server/server.go

```diff
@@ -44,3 +52,24 @@ func New(logger *slog.Logger, client ApiClient, templates map[string]*template.T
 
 	return otelhttp.NewHandler(http.StripPrefix(envVars.Prefix, securityheaders.Use(mux)), "supervision-finance-admin")
 }
+
+func getContext(r *http.Request) api.Context {
+	token := ""
+
+	if r.Method == http.MethodGet {
+		if cookie, err := r.Cookie("XSRF-TOKEN"); err == nil {
+			token, _ = url.QueryUnescape(cookie.Value)
+		}
+	} else {
+		token = r.FormValue("xsrfToken")
+	}
+
+	clientId, _ := strconv.Atoi(r.PathValue("clientId"))
```

Comment:
Why is this here? There is no "client" in Finance Admin.

Reply:
done